lib/master.js

```diff
@@ -302,14 +310,14 @@ function restart() {
   }
 
   function startOne() {
-    self.setSize(self.options.size + 1);
+    self.setSize(self.options.size + 1, true);
 
     self.once('resize', stopOne);
   }
 
   function stopOne() {
     waitForStability(function() {
-      self.setSize(self.options.size - 1);
+      self.setSize(self.options.size);
```

Comment:
There is a long comment on setSize, also stopOne is called from line 315 above, an in line 313, you can see the size was set to `self.options.size + 1`, so here we set the size back to `self.options.size`.